qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -211,6 +213,7 @@ def process_sample_template(self, study, user, callback):
                          else 'parsing the sample template')
             msg = html_error_message % (error_msg, basename(fp_rsp),
                                         str(e))
+            msg = convert_text_html(msg)
```

Comment:
I'm going to comment only here but applies to all. There is something weird when passing the `html_error_message` variable contents to the `convert_text_html`, as not all the html is interpreted correctly:
![screen shot 2015-08-07 at 2 06 42 pm](https://cloud.githubusercontent.com/assets/2501478/9145991/ded9a9bc-3d0d-11e5-9ba8-a0da0b645bc6.png)
Note that the only part of the error message that needs to be passed to `convert_text_html`, which is the variable `e`.
Suggested change:
```python
msg = html_error_message % (error_msg, basename(fp_rsp), convert_text_html(str(e)))
```